modules/k8s/tanka/new-app.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+APP_NAME=$1
+
+NAMESPACE=${NAMESPACE:-""}
+OWNER=${OWNER:-""}
+IMAGE_REPOSITORY=${IMAGE_REPOSITORY:-""}
+IMAGE_TAG=${IMAGE_TAG:-""}
+
+echo ""
+
+if [ -z ${APP_NAME} ]; then
+  echo "Must specify an application name"
+  exit 1
+fi
+if [ -d "lib/${APP_NAME}" ] || [ -d "environments/${APP_NAME}" ]; then
+  echo "Application already exists by the name '${APP_NAME}'"
+  exit 1
+fi
+
+if [ "${NAMESPACE}" = "" ]; then
+  namespaces=$(tk env list --json | jq -r '.[].spec.namespace' | sort | uniq)
+  if [ ${#namespaces[@]} == 1 ]; then
+    NAMESPACE="${namespaces[0]}"
+  else
+    echo "Namespace:"
+    read ns
+    NAMESPACE=$ns
+  fi
+fi
+
+if [ "${OWNER}" = "" ]; then
+  echo "Owner:"
```

Comment:
Maybe slugify this:

https://gist.github.com/oneohthree/f528c7ae1e701ad990e6#file-quick-slugify-sh

Ideally we have a pick list, but not sure where the source of truth is for that (probably not here)